stable/kibana/templates/deployment.yaml

```diff
@@ -12,8 +12,9 @@ spec:
   revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
   template:
     metadata:
-    {{- if .Values.podAnnotations }}
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+    {{- if .Values.podAnnotations }}
```

Comment:
can we outdent this line and L19? that's the more cohesive method of doing if|ends now and greatly enhances readability.

Reply:
Sure! Done.